src/document/DocumentManager.ts

```diff
@@ -705,6 +707,22 @@ export async function newProject() {
   uiState.setProjectSavingState(SavingState.NO_LOCATION);
   uiState.setProjectSavingTime(new Date());
   doc.history.clear();
+  if (!promptForCodegen) {
+    return;
+  }
+  const msg = codegenEnabled()
+    ? `
+    Change the folder in which java file generation occurs?
+    (Do this if you're switching to a new codebase)
+    `
+    : `
+    Choreo can now output java files
```

Comment:
```suggestion
    Choreo can now output Java files
```
Again, the `now` phrasing will age poorly because the feature won't be new forever. Please rephrase.